gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/installation/EnvironmentResource.java

```diff
@@ -66,6 +67,11 @@ public GroupsResource getGroupsResource() {
         return resourceContext.getResource(GroupsResource.class);
     }
 
+    @Path("/ui/portal-menu-links")
```

Comment:
I am wondering if eventually it would make sense to have `/portal/` in the url that can also apply to `/themes`, `/settings` for portal, and `/menu-links`.

Reply:
I agree. But it deserves more work for refactoring.